apps/api/src/hyp3_api/job_validation_map.yml.j2

```diff
@@ -0,0 +1,7 @@
+{% for job_spec in job_types.values() %}
+{{ job_spec['api_job_type'] }}: {% if not job_spec['validators'] %}[]{% endif %}
+
+  {% for validator in job_spec['validators'] %}
+  - {{ validator }}
+  {% endfor %}
```

Comment:
`job_spec['validators']` is a valid python list, could this all just be:
`{{ job_spec['api_job_type'] }}: {{ job_spec['validators'] }}` ?

I'd expect that to render as:
`foo: []`
`bar: [abc, def,]`